### PR TITLE
Bring Github Afkak up to parity with Ciena Afkak

### PR DESCRIPTION
**Pull request by rthille**

### CODE REVIEW

**Comment by rthille** [comment]
Grr, I can't assign to @twm for some reason...

**Comment by twm** [comment]
I'm going to go ahead and merge this, but we should follow-up on some of these review comments.

**Comment by rthille** [comment]
@twm What review comments?

File: afkak/client.py [hunk 1]
```diff
@@ -662,8 +669,13 @@ def _cancel_timeout(_, dc):
             # Set a delayedCall to fire if we don't get a reply in time
             dc = self._get_clock().callLater(
                 self.timeout, _timeout_request, broker, requestId)
-            # Setup a callback on the request deferred to cancel timeout
+            # Set a delayedCall to complain if the reactor has been blocked
+            rc = self._get_clock().callLater(
+                (self.timeout * 0.9), _alert_blocked_reactor, self.timeout,
+                time.time())
```
Comment (rthille) [inline afkak/client.py]: Is there a way to get "wall time" from the reactor?

Comment (twm) [inline afkak/client.py]: Given what you do with this, I don't think that you need wall clock time. The elapsed time you can get from `IReactorTime.seconds()` is fine.

I vaguely recall seeing Glyph say that `seconds()` actually should be wall clock time somewhere. And practically speaking it probably couldn't be changed without breaking code, so it's probably not worth worrying about here. The only place I've seen it not be wall-clock time is the Z-series.

Comment (rthille) [inline afkak/client.py]: Switching to `reactor.seconds()` might have made writing the test easier though. :-)  I'll have to go back and look at that test.

**Pull request merged**